packages/webapp/src/app/mini-app/page.tsx

```diff
@@ -2,66 +2,84 @@
 
 import { useLaunchParams, biometry } from "@telegram-apps/sdk-react";
 import { useEffect, useState } from "react";
+import { Client, networks } from "@/../../../.soroban/account/src";
+
+const greeter = new Client({
+  ...networks.testnet,
+  rpcUrl: process.env.NEXT_PUBLIC_SOROBAN_RPC_URL || "https://soroban-testnet.stellar.org", 
+});
 
 const MiniApp = () => {
     const launchParams = useLaunchParams();
     const [isLoading, setIsLoading] = useState<boolean>(true);
     const [error, setError] = useState<string | null>(null);
     const [userId, setUserId] = useState<number | null>(null);
     const [name, setName] = useState<string | null>(null);
+    const [greeting, setGreeting] = useState<string | null>(null);
 
     useEffect(() => {
         const initializeComponent = async () => {
-        try {
-            if (launchParams?.initData) {
-            const initData = launchParams.initData;
-            const uid = initData.user?.id;
-            const name = initData.user?.firstName + " " + initData.user?.lastName;
-            setUserId(uid ?? null);
-            setName(name);
-
-            // Check if biometry is available and mount it
-            if (biometry.mount.isAvailable()) {
-                try {
-                await biometry.mount();
-                if (biometry.isMounted()) {
-                    // Attempt to authenticate the user
-                    if (biometry.authenticate.isAvailable()) {
-                    const { status, token } = await biometry.authenticate({
-                        reason: 'Please authenticate to continue',
-                    });
-
-                    if (status === 'authorized') {
-                        console.log(`Authorized. Token: ${token}`);
+            try {
+                if (launchParams?.initData) {
+                    const initData = launchParams.initData;
+                    const uid = initData.user?.id;
+                    const name = initData.user?.firstName + " " + initData.user?.lastName;
+                    setUserId(uid ?? null);
+                    setName(name);
+
```

Comment:
You added demo code for the contract that will be overwritten by the next PR. What is the reason for it? Please rollback the entire file